### PR TITLE
fix: testing type configured from initial config

### PR DESCRIPTION
**Pull request by estrada9166**
- Closes  [UNIFY-936](https://cypress-io.atlassian.net/browse/UNIFY-936)

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/app/cypress/e2e/sidebar_navigation.cy.ts [hunk 1]
```diff
@@ -1,176 +1,212 @@
...
+
+      cy.findByLabelText('Sidebar').closest('[aria-expanded]').should('have.attr', 'aria-expanded', 'false')
+
+      cy.get('[data-cy="keyboard-shortcuts"]').should('be.visible')
+      cy.get('[data-cy="keyboard-shortcuts"]').click()
+      cy.contains('h2', 'Keyboard Shortcuts').should('be.visible')
+      cy.get('li p').contains('Re-run tests').should('be.visible')
+      cy.get('li p').contains('Stop tests').should('be.visible')
+      cy.get('li p').contains('Toggle specs list').should('be.visible')
+      cy.get('li span').contains('r')
```
Comment (lmiller1990) [inline packages/app/cypress/e2e/sidebar_navigation.cy.ts:57]: What is `r` `s` and `f` here? This part test is a little confusing.

**Review by lmiller1990 — COMMENTED** [review]
I am not sure how I'm meant to test this - I tried doing what I thought the ticket said, but I ran into this bug: https://cypress-io.atlassian.net/browse/UNIFY-936?focusedCommentId=15711 is this related?

File: packages/data-context/src/actions/ProjectActions.ts [hunk 2]
```diff
@@ -370,6 +370,7 @@ export class ProjectActions {
 
   async reconfigureProject () {
     await this.api.closeActiveProject()
+    await this.initializeActiveProject()
```
Comment (lmiller1990) [inline packages/data-context/src/actions/ProjectActions.ts:373]: Seems you can remove line 372 if you like - `initializeActiveProject` does `closeActiveProject` for you.

https://github.com/cypress-io/cypress/blob/10.0-release/packages/data-context/src/actions/ProjectActions.ts#L123

**Review by lmiller1990 — DISMISSED** [review]
One comment, but code seems fine :+1:

**Review by lmiller1990 — APPROVED** [review]

**Comment by lmiller1990** [comment]
lgtm!!

**Comment by estrada9166** [comment]
closing in favor of:
- #19904

**Pull request closed without merging**